.github/workflows/ci.yaml

```diff
@@ -94,13 +94,20 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Install system dependencies
-        run: sudo apt-get install -y graphviz rsync curl
+        run: sudo apt-get install -y curl git graphviz rsync
 
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip packaging setuptools twine
           pip install --upgrade -e .[all]
 
+      - name: Set up Git
+        run: |
+          # From: https://stackoverflow.com/questions/62960533/how-to-use-git-commands-during-a-github-action
+          git --version
+          git config --global user.email "test@bsc.es"
+          git config --global user.name "GitHub Actions"
+
```

Comment:
Integration tests now run `git` to create a fake local Git project. So we need to install `git` (above) and also configure the identity info.